Dev server crashes when file is renamed

On OS X, watchman 4.7.0, I get a crash whenever a data file is renamed:

```
ERROR in ./src/structure/urls.js
Module not found: Error: Can't resolve 'content/structure/navigation.yml' in '/Users/me/code/Go/src/github.com/Client/www/frontend/packages/project-www-website/src/structure'
 @ ./src/structure/urls.js 14:18-61
 @ ./src/structure/Routes.js
 @ ./src/index.js
 @ multi (webpack)-hot-middleware/client.js ./~/react-hot-loader/patch.js ./src/index.js
webpack: Failed to compile.
events.js:160
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot read property 'name' of undefined
    at /Users/me/code/Go/src/github.com/Client/www/frontend/packages/project-www-website/node_modules/@phenomic/core/lib/watch/index.js:120:18
    at Array.find (native)
    at WatchmanWatcher.<anonymous> (/Users/me/code/Go/src/github.com/Client/www/frontend/packages/project-www-website/node_modules/@phenomic/core/lib/watch/index.js:119:23)
    at emitThree (events.js:116:13)
    at WatchmanWatcher.emit (events.js:194:7)
    at WatchmanWatcher.emitEvent (/Users/me/code/Go/src/github.com/Client/www/frontend/packages/project-www-website/node_modules/sane/src/watchman_watcher.js:266:8)
    at WatchmanWatcher.handleFileChange (/Users/me/code/Go/src/github.com/Client/www/frontend/packages/project-www-website/node_modules/sane/src/watchman_watcher.js:227:10)
    at Array.forEach (native)
    at WatchmanWatcher.handleChangeEvent (/Users/me/code/Go/src/github.com/Client/www/frontend/packages/project-www-website/node_modules/sane/src/watchman_watcher.js:189:16)
    at emitOne (events.js:96:13)
    at Client.emit (events.js:188:7)
error Command failed with exit code 1.
```

--- a/packages/core/src/watch/index.js
+++ b/packages/core/src/watch/index.js
@@ -100,7 +100,7 @@ function createWatcher(config: { path: string, plugins: PhenomicPlugins }) {
   watcher.on("delete", (filepath, root) => {
     debug("watcher: file deleted", filepath, root);
     const index = files.find(
-      (file: PhenomicContentFile) => file.name === filepath
+      (file: PhenomicContentFile) => file && file.name === filepath
     );
     if (index) {
       delete files[files.indexOf(index)];

